deployments/cdi/systemd/cdi-spec.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Creates CDI spec file
+# Removal of this file signals firstboot completion
+# This service is used to configure the SR-IOV VFs on NICs
+Wants=network-pre.target
+Before=network-pre.target
```

Comment:
why it network-pre.target ?
should it be after network so that we will be sure that the driver is loaded?

Reply:
according to https://systemd.io/NETWORK_ONLINE/: 

> network-pre.target is used to order services before any network interfaces start to be configured. Its primary purpose is for usage with firewall services that want to establish a firewall before any network interface is up.